templates/_admin_configure_module.tpl

```diff
@@ -7,6 +7,18 @@
             {% wire id="apikey" type="blur" action={config_toggle module="mod_geo" key="api_key" on="keyup"} %}
         </div>
     </div>
+    <div class="control-group">
+        <label class="control-label" for="startingpoint">{_ Google maps starting point _}</label>
+        <div class="controls">
+            {% if m.config.mod_geo.api_key.value %}
+                <div id="{{ #lazy }}">
+                    {% lazy action={update target=#lazy id=id template="_admin_configure_map.tpl"}%}
+                </div>
+            {% else %}
+                {_ First add the Google maps API key _}
+            {% endif %}
```

Comment:
I would suggest hiding this block (control-group) completely until the Google Maps API key is filled in, as this looks a bit weird:

![screen shot 2017-05-29 at 12 18 39](https://cloud.githubusercontent.com/assets/89267/26546185/f78e86b2-4468-11e7-8bac-f57ac1f8ced4.png)

Then add a wire to the API key so that when that is filled in, this block is automatically shown. Otherwise you need to close and re-open the configuration dialog before being able to add a start location.